arduino-ide-extension/src/node/arduino-localization-contribution.ts

```diff
@@ -0,0 +1,150 @@
+import {
+  LocalizationContribution,
+  LocalizationRegistry,
+} from '@theia/core/lib/node/i18n/localization-contribution';
+
+export class ArduinoLocalizationContribution
+  implements LocalizationContribution
+{
+  async registerLocalizations(registry: LocalizationRegistry): Promise<void> {
+    registry.registerLocalizationFromRequire(
+      'af',
+      require('../data/i18n/af.json')
```

Comment:
The path `arduino-ide-extension/lib/node/data/i18n` will not exist in your app, so this require call will result in an error. You need to point these calls to the correct path of your localization files.